package.json

```diff
@@ -5,6 +5,7 @@
   "author": "salomonelli",
   "scripts": {
     "dev": "node build/dev-server.js",
+    "build": "node build/build.js",
```

Comment:
maybe include a deploy script here which will run gh-pages
also maybe add gh-pages to devdeps ?

Reply:
it's a good suggestion, although travis will deploy to GitHub Pages every time you push a commit